1-exercises/A-reading-list/script.js

```diff
@@ -1,27 +1,39 @@
 function readingList(books) {
-  // Write your code here...
+  let ulEl = document.createElement("ulEl");
```

Comment:
Typo, should be **ul**

Reply:
Of course, sorry I completely forgot it.